wrappers/wrapper.rb

```diff
@@ -61,9 +61,12 @@ def assert_vehicles_no_capacity_initial(vrp)
     end
 
     def assert_vehicles_no_alternative_skills(vrp)
-      vrp.vehicles.empty? || vrp.vehicles.none?{ |vehicle|
-        !vehicle.skills || vehicle.skills.size > 1
-      }
+      vrp.vehicles.none?{ |vehicle| vehicle.skills.size > 1 }
+    end
+
+    def assert_vehicles_no_alternative_skills_with_periodic_heuristic(vrp)
+      !vrp.preprocessing_first_solution_strategy.include?('periodic') ||
+        vrp.vehicles.none?{ |vehicle| vehicle.skills.size > 1 }
     end
```

Comment:
Can't we move this assert to vrp.create? I feel like in this case we refuse to solve the VRP, right? So there is no need to create the VRP.